### DISCUSSION TITLE
Scrollbar is not stopping at the bottom of the view

### DISCUSSION THREAD

**Discussion by njust** [lapce/floem#429] [Q&A] [answered]
Hi,
I'm trying to get the left sidebar example working without absolute positioning of the content. But for some reason, after changing the layout, the scrollbar is not stopping at the bottom of the view:

https://github.com/lapce/floem/assets/2380253/5807b67e-03aa-424e-be6f-2301a595a922

Here is the example repository:
https://github.com/njust/floem/tree/scrolling_issue

Any ideas what I'm doing wrong here? Is #428 somehow related?

**Comment by jrmoulton** [comment]
This looks like you are using a virtual list. When using a virtual list you need to tell the virtual list the size of each element. If the scroll bar isn't ending at the bottom it's probably because you are passing in the incorrect height to the virtual list

**Reply by njust** [reply]
I'm talking about the official example:
![image](https://github.com/lapce/floem/assets/2380253/2a976e8c-6bd6-4e0c-b248-362758852f96)

My changes to the example are minimal:
https://github.com/lapce/floem/compare/main...njust:floem:scrolling_issue

Even when replacing the virtual_stack in the offical example with a v_stack, the problem remains the same.

**Reply by dominikwilkowski** [reply, marked as answer]
Just found this. Note that this example was fixed in #430
The height must have shifted since I wrote this example and now I explicitly fix the height for each item which makes sure this works even if things shift further down the line.

You can see the height of each item was was off by one pixel with the inspector btw. So if this happens again in your own code just inspect the item and make sure what you tell the `v_stack` or `virtual_list` is the correct height.

**Reply by njust** [reply]
Thanks!
